src/views/errors/session-ended.njk

```diff
@@ -1,4 +1,16 @@
 {% extends "base-page.njk" %}
+{% from "components/onPageLoad/macro.njk" import ga4OnPageLoad %}
 
 {% set hmpoPageKey = "sessionEnded" %}
 {% set gtmJourney = "kbv-hmrc - error" %}
+
+{{ ga4OnPageLoad({
+  nonce: cspNonce,
+  statusCode: '400',
+  englishPageTitle: 'error.sessionEnded.title' | translate,
```

Comment:
Can we use `hmpoPageKey` here (and for all error pages) ?

Reply:
hmpoPageKey is object, may be hmpoTitle we can use.